stdlib/public/TensorFlow/Dataset.swift

```diff
@@ -19,12 +19,7 @@
 /// A `SingleValueDataset` can be used to represent an input pipeline as a
 /// collection of element tensors.
 @_fixed_layout
-public struct SingleValueDataset {
-  // NOTE: SingleValueDataset is currently specialzed on `Float` to work around
-  // deabstraction bug SR-8463. This is to avoid indirect passing in SIL. When
-  // the bug gets resolved, `SingleValueDataset` will have generic parameter
-  // `Scalar`.
-  public typealias Scalar = Float
+public struct SingleValueDataset<ScalarOfElement : AccelerableByTensorFlow> {
```

Comment:
Out of curiosity, why not just Scalar or Element?  ScalarOfElement is sort of a weird name.

Reply:
Oh sorry, I see bgogul asked the same question below. n/m!